_drafts/contribute-to-ansible.markdown

```diff
@@ -2,11 +2,11 @@
 
 ## Introduction
 
-    voici un article blablabla ....
+    Because we find Ansible, just awsome, we would like to share our experience about contributing to it. We will see, step-by-step, how to contribute to Ansible and generally on a lot of open-source projects. We so will begin it by refreshing you on what is Ansible. Next we will see four steps, how to set-up your environment, how to develop on it, how to perform your first pull-request and then we will se the best process to have your request accepted. 
 
 ## Goals
 
-    become Ansible contributor
+    The goal of this article is, of course, how to become an Ansible commiter, but not only, we will help you to speed up your pull-request merge ! Ansible is needing help, and of course everyone are welcome ! Let's begin !
```

Comment:
commiter? contributor instead of commiter.
commiter is therm dedicated to ansible official core development team.
Remove space between letters and !

Reply:
GFY !